files/lib/gluon/v14tov15-helper/v14tov15-helper

```diff
@@ -1,105 +1,108 @@
 #!/bin/sh +x
 
-# script to help meshnodes make an autoupdate
-# a node which looses connection (because a uplink mesh node is talking newer 
-# batman v15) try to log in as client in a network with well known essid
-# and forces a autoupdate (if it is turned on in general)
+# this script will help meshnodes to make an autoupdate when a node looses connection 
+# because all uplink mesh nodes are talking an incompatible mesh protocol, for example  
+# batman v15. It will try to log in as a client into a network with well known essid
+# and force an autoupdate (if autoupdate is turned on in general)
 #
-# this script do not do any harm if running for years, we suggest running once a day
+# this script doesn't do any harm if running for years, we suggest running it once a day
 # more Info: https://github.com/viisauksena/gluon-v14tov15-helper
 #
-# maybe you have to adjust nameservers, etc/hosts entry and fallback-ffssid
+# maybe you have to adjust the nameservers, etc/hosts entry and fallback-SSID
 # below
 
-# some cron debug help , write in ram 
-echo "$0 successfully start at $(date +%y%m%d_%H%M)" > /tmp/v14_lastrun
+# fallback-SSID
+# ffssid=$(uci get wireless.client_radio0.ssid) # fail with some ssid offline changers
+: ${ffssid:=freiburg.freifunk.net} # if for whatever reason ffssid is NULL, obviously not generic
+
+# nameservers 
+NS1=10.60.0.100
+NS2=4.2.2.2
+NS3=10.60.0.110
```

Comment:
ok, i may look into it - but in general i had issues to make this package more generic - and  i have to think deeper into it - i may read (and write if not there) some uci stuff, so ever community could change it by its own